entsoe-balances-adjustment/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>powsybl-entsoe</artifactId>
+        <groupId>com.powsybl</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>powsybl-entsoe-balances-adjustment</artifactId>
```

Comment:
FIXME here name and description. For the artifactId, I think that you have to explicitly says that we do that for CGMES-based process. And finally maybe the "other" control area based on `CgmesControlArea` extension has to be moved here too.

Reply:
I'm modifying it as `powsybl-entsoe-cgmes-balances-adjustment`